docs/GSMTC-Support-And-Popular-Apps.md

```diff
@@ -22,26 +22,27 @@ So, let's check the compatibility or support report.
 - 🟪 - Feature support is incomplete i.e. in-progress or being improved or expected to arrive soon or temporarily unavailable
 - 🟥 - Feature not supported
 
-| Application | Support | Play\Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
+| Application | Support | Play/Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
 | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
 | **Browsers** |
-| Chromium and its derivatives (such as **Chrome**, **New MS Edge**, **Opera**, **Brave**, **Vivaldi** and etc.) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
+| Chromium and derivatives (Chrome, New MS Edge, Opera, Brave, Vivaldi, etc.) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
 | Firefox | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
 | **Media players** |
-| *(i) Music Only* |
+| **M** |**U**|**S**|**I**|**C**||**O**|**N**|**L**|**Y**||
```

Comment:
A big no for this. It would be easier to have the text combined. It's too distracting now

Reply:
Yeah, that's why I wanted to ask about this.

Reply:
Can we remove these distinctions all together? The audio only, video only, and both ones?

Reply:
It would help segregate the clutter. So, let's keep them but not like that spacy like you changed.

Reply:
We can use the h1, h2, h3 tags here in the table, right? How about using them?